### PR TITLE
#379 Remove deprecated property use

### PR DESCRIPTION
**Pull request by IRus**

### CODE REVIEW

File: src/main/groovy/net/researchgate/release/ReleasePlugin.groovy [hunk 1]
```diff
@@ -104,7 +104,6 @@ class ReleasePlugin extends PluginHelper implements Plugin<Project> {
             startParameter.projectProperties.putAll(project.getGradle().startParameter.projectProperties)
             startParameter.projectProperties.put('release.releasing', "true")
             startParameter.projectDir = project.projectDir
-            startParameter.settingsFile = project.getGradle().startParameter.settingsFile
```
Comment (Hillkorn) [inline src/main/groovy/net/researchgate/release/ReleasePlugin.groovy:107]: Please wrap this in an if that checks the gradle version to not break existing usages of it.

Comment (IRus) [reply src/main/groovy/net/researchgate/release/ReleasePlugin.groovy:107]: @Hillkorn updated

**Review by Hillkorn — APPROVED** [review]

File: src/main/groovy/net/researchgate/release/ReleasePlugin.groovy (hunk 1 above)
Comment (Hillkorn) [reply src/main/groovy/net/researchgate/release/ReleasePlugin.groovy:107]: Thanks

**Comment by oallauddin** [comment]
@Hillkorn Could you cut a release?

**Pull request merged**